### ISSUE TITLE
Minor inconsistency in menu items

### ISSUE THREAD

**Issue by Ungoose** [osmandapp/OsmAnd#11004]
### Description
Not a bug but rather a minor nagging issue with inconsistent UI. All of the options in the drawer use sentence case (only first word is capitalized) except for My Places, which capitalizes both for some reason (OsmAnd Live is a proper noun so no questions there). Also, Plan a route is the only option that uses an article. Why not change it to "Plan route" to make the menu more consistent? Thanks!

### How to reproduce?
Open drawer from any profile

### Your Environment
OsmAnd Version: 3.9.9
Android/iOS version: any
Device model: any

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by Ungoose** [comment]
So I've checked the strings.xml and it appears that there are two similar strings: "Plan a route" at 316 and "Plan route" at 1432. Will there be any harm in changing the name of the first one to `<string name="plan_a_route">Plan route</string>` or will it mess up the second one?

**Comment by dmpr0** [comment]
We need to:
- Change "My Places" to "My places"
- For "Plan a route" use already existing string https://github.com/osmandapp/OsmAnd/blob/master/OsmAnd/res/values/strings.xml#L1433

**Comment by Ungoose** [comment]
> 
> 
> We need to:
> 
>     * Change "My Places" to "My places"
> 
>     * For "Plan a route" use already existing string https://github.com/osmandapp/OsmAnd/blob/master/OsmAnd/res/values/strings.xml#L1433

For some reason, the latest night build has My Places capitalized again (plan route is fine, though).

**Comment by sonora** [comment]
For what it's worth (If you find this hair-splitting, just smile ;) ):

- The activities "Configure screen" and "Configure map" in the menu refer to **definite targets**, so what is left out here is the definite article ("Configure _the_ screen", "Configure _the_ map"). Inserting it would not create any more clarity.

But these are **indefinite**:
- "Download maps": (You can chose which maps.) There is no plural indefinite article in English
- "Plan a route": It's indefinite, you can plan any route you wish to.

"Plan route" sounds a bit off, like we either try desperate to save the indefinite "a", or may even mean to refer the something speific "Plan the route", which we do not.

**Long story short: While it's no big deal, "Plan a route" seems correct.**

Regarding "My Places", this was originally capitalized because it refers to a _specific app area_ (the "My Tracks" screen and underlying object collections like your favorites and tracks), while "my places" is a general and unspecific term not indicating it when it means to refer to that repository in the app. But this decision point is a lot finer, at least ever since we also decided to refer to the "favorites" (originally capitalized for the same reason) section in lower case now in our documentation.

**Comment by vshcherb** [comment]
Yes, the article is needed.
